pkg/operator/ceph/object/user/controller_test.go

```diff
@@ -412,7 +412,7 @@ func TestCreateOrUpdateCephUser(t *testing.T) {
 						StatusCode: 200,
 						Body:       io.NopCloser(bytes.NewReader([]byte(userCreateJSON))),
 					}, nil
-				} else if req.URL.RawQuery == "caps=&format=json&uid=my-user&user-caps=users%3Dread%3Bbuckets%3Dread%3B" {
+				} else if req.URL.RawQuery == "caps=&format=json&uid=my-user&user-caps=users%3Dread%3Bbuckets%3Dread%3Brole%3Dread%3B" {
```

Comment:
I'm okay with just testing one of the new ones since we can see they're all the same here. But I think it would be good to see that `read, write` and `*` get captured correctly when queried from the RGW. Do we know what the RGW returns in that case?